### ISSUE TITLE
Invalid escape sequences warn when building in Python 3.8

### ISSUE THREAD

**Issue by tirkarthi** [workhorsy/py-cpuinfo#138]
Using raw strings or escaping them will fix the issue.

```
find . -iname '*.py'  | xargs -P 4 -I{} python3.8 -Wall -m py_compile {}

./tools/get_system_info.py:90: DeprecationWarning: invalid escape sequence \d
  if re.match('^i\d86$|^x86$|^x86_32$|^i86pc$|^ia32$|^ia-32$|^bepc$', arch_string_raw):
./cpuinfo/cpuinfo.py:466: DeprecationWarning: invalid escape sequence \(
  starts = [m.start() for m in re.finditer('\(', cpu_string)]
./cpuinfo/cpuinfo.py:467: DeprecationWarning: invalid escape sequence \)
  ends = [m.start() for m in re.finditer('\)', cpu_string)]
./cpuinfo/cpuinfo.py:616: DeprecationWarning: invalid escape sequence \d
  if re.match('^i\d86$|^x86$|^x86_32$|^i86pc$|^ia32$|^ia-32$|^bepc$', arch_string_raw):
./tests/test_linux_ubuntu_16_04_x86_64.py:127: DeprecationWarning: invalid escape sequence \_
  output = '''
./tests/test_linux_debian_8_5_x86_64.py:89: DeprecationWarning: invalid escape sequence \_
  output = '''
./tests/test_linux_fedora_24_x86_64.py:89: DeprecationWarning: invalid escape sequence \_
  output = '''
```

**Comment by workhorsy** [comment]
Thanks. I'll see if I can update the test suite to test this automatically when run.

**Comment by tirkarthi** [comment]
Using `-Werror` or the environment variable `PYTHONWARNINGS=error` helps but all deprecation are turned into error. Flake8 might help since it has a rule to detect this.

**Comment by workhorsy** [comment]
Fixed in #148. The test suite should test this automatically. Let me know if you have any more problem. Thanks
